lib/etc/openturns.conf.in

```diff
@@ -213,19 +214,19 @@
   <Compact-DefaultHalfMaximumSize value_int="1024" />
 
   <!-- OT::FaureSequence parameters -->
-  <FaureSequence-InitialSeed value_int="1" />
+  <FaureSequence-InitialSeed value_int="0" />
 
   <!-- OT::HaltonSequence parameters -->
-  <HaltonSequence-InitialSeed value_int="1" />
+  <HaltonSequence-InitialSeed value_int="0" />
 
   <!-- OT::HaselgroveSequence parameters -->
-  <HaselgroveSequence-InitialSeed value_int="1" />
+  <HaselgroveSequence-InitialSeed value_int="0" />
 
   <!-- OT::ReverseHaltonSequence parameters -->
-  <ReverseHaltonSequence-InitialSeed value_int="1" />
+  <ReverseHaltonSequence-InitialSeed value_int="0" />
 
   <!-- OT::SobolSequence parameters -->
-  <SobolSequence-InitialSeed value_int="1" />
+  <SobolSequence-InitialSeed value_int="0" />
```

Comment:
Two questions, two answers:
+ no, I cannot convince the Sobol sequence to start at 0 by setting the initial value of the seed to 0 (any clue on how to fix the Sobol sequence is welcomed!). What you see is just a garbage of my failed attempts to solve it, thanks for catching it!
+ yes, openturns.conf is usefull as it allows to overwrite the default values hard-coded into the C++ by values more suited by an application relying on OT, for example the Anatole tolerancing workbench in CATIA. But this file has to be found by the library at startup in order to be taken into account, otherwise this is the hard-coded values that are taken into account. The rule is to have the exact same values as the hard-coded ones in this file by default, and if the user change them he has to make sure his local installation of OT find the file at startup.